### PR TITLE
[MODEBSNET-63] - Release v2.1.0

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose

https://issues.folio.org/browse/MODEBSNET-63
## Approach

image
image

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by SerhiiNosko — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
-## 2.1.0 - Unreleased
+## 2.2.0 - Unreleased
+
+## 2.1.0 - Released (Poppy R2 2023)
+
+The focus of this release was to update dependencies and remove unnecessary code
+
```
Comment (Abdulkhakimov) [inline NEWS.md:6]: Remove this line

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
-## 2.1.0 - Unreleased
+## 2.2.0 - Unreleased
+
+## 2.1.0 - Released (Poppy R2 2023)
+
```
Comment (Abdulkhakimov) [inline NEWS.md:4]: Remove this line

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+The focus of this release was to update dependencies and remove unnecessary code
+
+[Full Changelog](https://github.com/folio-org/mod-orders/compare/v2.0.0...v2.1.0)
+
+### Stories
+
```
Comment (Abdulkhakimov) [inline NEWS.md:10]: Also from here

File: NEWS.md [hunk 4]
```diff
@@ -1,4 +1,24 @@
...
+* [MODEBSNET-65](https://issues.folio.org/browse/MODEBSNET-65) spring boot upgrade
+* [MODEBSNET-64](https://issues.folio.org/browse/MODEBSNET-64) Folio Spring base updated
+* [MODEBSNET-62](https://issues.folio.org/browse/MODEBSNET-62) Delete unused Kafka and database references
+* [FOLIO-3678](https://issues.folio.org/browse/FOLIO-3678) Use GitHub Workflows api-lint and api-schema-lint and api-doc
+* [MODEBSNET-59](https://issues.folio.org/browse/MODEBSNET-59) Migrate to folio-spring-support v7.0.0
+* [MODEBSNET-58](https://issues.folio.org/browse/MODEBSNET-58) Delete beginFolioExecutionContext from TestBase
+
+### Dependencies
```
Comment (Abdulkhakimov) [inline NEWS.md:18]: Is it new requirement or agreement?

Comment (azizbekxm) [reply NEWS.md:18]: not requirement, looks good info, I read from documentation 
https://wiki.folio.org/pages/viewpage.action?pageId=96404780 
image
this info once help me to fix one issue.

File: NEWS.md (hunk 3 above)
Comment (azizbekxm) [reply NEWS.md]: Removed

File: NEWS.md (hunk 4 above)
Comment (Abdulkhakimov) [reply NEWS.md:16]: thanks, lgtm

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**